src/util/util_ext.c

```diff
@@ -240,3 +240,133 @@ errno_t sss_filter_sanitize(TALLOC_CTX *mem_ctx,
 {
     return sss_filter_sanitize_ex(mem_ctx, input, sanitized, NULL);
 }
+
+enum sss_trim_dn_state {
```

Comment:
This enum is used only in sss_trim_dn() so maybe it is a good idea to make it local for this function?

Reply:
done